src/PWinObj.hh

```diff
@@ -145,6 +145,8 @@ public:
     inline bool isFocusable(void) const { return _focusable; }
     //! @brief Returns true if the PWinObj is interested in keyboard input
     inline bool isKeyboardInput(void) const { return _keyboard_input; }
+    //! @brief Returns fullscreen state of PWinObj.
```

Comment:
I kind of think it's better to drop these comments, they don't add much information (I know I added the others in the first place)